src/services/game.ts

```diff
@@ -40,19 +40,29 @@ export const getGame: (type: number) => Promise<Game> = async (type: number) =>
 };
 
 interface ResultEntity {
-  readonly correct: number
-  readonly total: number
+  readonly currentScore: {
+    readonly count: number;
+    readonly time: number;
+  }
+  readonly previousBestScore: {
+    readonly time: number;
+  }
+  readonly betterScoresInLeaderboard: number;
 }
 
-interface Result {
+export interface GameResult {
   readonly score: number;
-  readonly count: number;
+  readonly time: number;
+  readonly rank: number;
+  readonly bestTime: number;
 }
 
-export const getScore: (gameId: string, answers: number[]) => Promise<Result> = async (gameId: string, answers: number[]) => {
-  const { data: { correct, total } } = await axios.post<ResultEntity>(`/games/${gameId}/score`, { answers: answers });
+export const getScore: (gameId: string, answers: number[]) => Promise<GameResult> = async (gameId: string, answers: number[]) => {
+  const { data: { currentScore, previousBestScore, betterScoresInLeaderboard } } = await axios.post<ResultEntity>(`/games/${gameId}/score`, { answers: answers });
```

Comment:
Selon quand ca va être mergé, ca pourrait ne plus être d'actualité (cf https://github.com/xebia-france/who-is-2-turbo-services/pull/29/files#diff-6da338a05bc7cbe4319698f78c56ece9fa395622d8ee3d934c52c8a5156f46d8)
Le endpoint `/score` va renvoyer 
```
      correct: seriesScore.score.count,
      total: seriesScore.solutions.length,
      solutions: seriesScore.solutions,
      rank: seriesScore.rank,
      score: seriesScore.score,
      bestRank: seriesScore.bestRank,
      bestScore: seriesScore.bestScore,
```

Du coup, à changer par (sauf erreur de ma part)
```
const { data: { score, bestScore, bestRank } } = await axios.post<ResultEntity>(`/games/${gameId}/score`, { answers: answers });

return {
    score: score.count,
    time: Math.round(score.time / 1000),
    rank: bestRank + 1,
    bestTime: Math.round(bestScore.time / 1000),
  };
```

Reply:
Oui, c'est fait, il faudra attendre de fusionner la PR https://github.com/xebia-france/who-is-2-turbo-services/pull/29 je vais prendre en compte les retours.